### DISCUSSION TITLE
Any way to integrate changes made to a module/pod into the toolchain?

### DISCUSSION THREAD

**Discussion by theabhishek2511** [ionic-team/capacitor#6962] [Q&A] [answered]
I've had to make a few changes in the native code in the pods of an old project I've been maintaining to make it work, specifically in ```HttpRequestHandler.swift```. However I'm aware that if pods are refreshed I might lose those changes. Is there a way to work said changes into the toolchain? kind of like how ```patch-package``` works.

**Comment by jcesarmobile** [comment, marked as answer]
Best way is to not do changes in a module/pod.

If you really really need to, patch-package should work since Capacitor and Capacitor plugins are npm packages and their source comes from npm and the source code is in node_modules folder.
